tech/languages/r/r-packages.md

```diff
@@ -0,0 +1,69 @@
+---
+title: R packages
+subsection: r
+order: 2
+---
+
+# Installing packages
+
+Recommended R packages are included as part of the 'R-core' component.
+A number of popular add-on packages from [CRAN](https://cran.r-project.org/), [Bioconductor](https://bioconductor.org/) and other sources are readily available via the official repositories.
+They are named with an `R-` prefix, such as `R-ggplot2`. Hence, running
+
+```bash
+$ dnf repoquery --repo=fedora-source R-*
+```
+
+provides a comprehensive list of packages.
+
+## Installation from source
+
+If the 'R' meta-package is installed, thousands of additional add-on packages can be installed from the official CRAN and Bioconductor repositories.
+To install a package from source, open the _R console_ (see [Running R](/tech/languages/r/r-installation.html#running-r)) and type the following (e.g., for the `ggplot2` package):
+
+```r
+install.packages("ggplot2")
+```
+
+With this method, packages are installed into the user home directory, under `~/R`.
+
+See also the [`remotes`](https://remotes.r-lib.org/) package, to learn how to install packages from a variety of sources (e.g., GitHub),
+and the [`renv`](https://rstudio.github.io/renv/) package, to learn how to manage environments and to install specific versions of packages.
+
+## Additional binary packages
+
+The [cran2copr](https://copr.fedorainfracloud.org/coprs/iucar/cran/) project maintains binary RPM contributed repositories for the current and previous stable Fedora version for most of CRAN (~17k packages as of Apr 2021) in an automated way using [Fedora Copr](https://copr.fedorainfracloud.org/).
+
+These repositories are automatically synchronized with CRAN every day at 00:00 UTC through a GitHub Action that removes archived packages and builds the most recent updates.
```

Comment:
1. Licenses are correct because CRAN licenses are compatible with Fedora licenses, and CRAN has strict checks for them.
2. I think that the fact that the repo is up-to-date is important for the R user.

Reply:
I removed the technical details, but I think it's important to say that it's updated on a daily basis.

Reply:
Oh, and when I say "CRAN has strict checks", I mean that there is actually a human being checking the license and the code for every new package accepted on CRAN.

Reply:
>     1. Licenses are correct because CRAN licenses are compatible with Fedora licenses, and CRAN has strict checks for them.

> Oh, and when I say "CRAN has strict checks", I mean that there is actually a human being checking the license and the code for every new package accepted on CRAN.


Cool, very cool.

> 
>     2. I think that the fact that the repo is up-to-date is important for the R user.

> I removed the technical details, but I think it's important to say that it's updated on a daily basis.

Well, I'm afraid think users don't care that much :). But it's fine with me if you leave a note there that the COPR repo is always up-to-date, or such - it's _nice to have_. I simply wouldn't go into much detail (don't overload the user!).

Reply:
Ok, this is done in https://github.com/developer-portal/content/pull/374/commits/20ff3d898bbf404667a0882ae8989c915c33403e